src/ecdsa/test_pyecdsa.py

```diff
@@ -230,6 +237,19 @@ def test_privkey_strings(self):
         priv2 = SigningKey.from_der(s1)
         self.assertTruePrivkeysEqual(priv1, priv2)
 
+        priv1 = SigningKey.generate(curve=BRAINPOOLP512r1)
+        s1 = priv1.to_pem()
+        self.assertEqual(type(s1), binary_type)
+        self.assertTrue(s1.startswith(b("-----BEGIN EC PRIVATE KEY-----")))
+        self.assertTrue(s1.strip().endswith(b("-----END EC PRIVATE KEY-----")))
+        priv2 = SigningKey.from_pem(s1)
+        self.assertTruePrivkeysEqual(priv1, priv2)
+
+        s1 = priv1.to_der()
+        self.assertEqual(type(s1), binary_type)
+        priv2 = SigningKey.from_der(s1)
+        self.assertTruePrivkeysEqual(priv1, priv2)
+
```

Comment:
this is not using any values generated by previous code, couldn't it be a separate test case?

Reply:
They are part of the same set of tests, just with a different curve, isn't it?

Reply:
yes, but that means if the test of the NIST256p fails, this code doesn't run
so if a change breaks both, curves, the report will not provide that information

Reply:
got it!